test_files/preprocessor.h

```diff
@@ -0,0 +1,4 @@
+
+#define FOO 0
+#define FOOBAR 1
+#define FOOBARBAZ 2
```

Comment:
Yes, I have in plans to improve the handling of `#define`s, but lets make it step by step.